### ISSUE TITLE
Support High Refresh Rate 

### ISSUE THREAD

**Issue by yoasif** [mozilla-mobile/android-components#6493]
Initially reported in https://github.com/mozilla-mobile/fenix/issues/7560

Steps to reproduce

    Open testufo.com
    Set "Count of UFOs" to 1

Expected behavior

Get 90FPS on a 90Hz Screen
Actual behavior

Getting 60FPS while Chrome gets 90

--

This may be supported using the frame pacing library:

https://developer.android.com/games/sdk/frame-pacing
https://source.android.com/devices/graphics/frame-pacing

It is probably also worth looking into how upstream Chromium achieves this, since high refresh rate displays will likely go up in use.

**Comment by mugna91** [comment]
Any update on this?

**Comment by dead10ck** [comment]
As a user of a high refresh rate device, I really hope this gets done before the new UI is released on the standard version of Firefox for Android.

**Comment by rosuH** [comment]
I am using op8 and I found that the issues are caused by onePlus. They have a whitelist for supporting the high refresh rate feature.

See here: https://bugzilla.mozilla.org/show_bug.cgi?id=1662380

**Comment by ReeganExE** [comment]
Any plan for this?

**Comment by dead10ck** [comment]
I haven't had any issues with the refresh rate since the new UI went stable. I think it was a OnePlus-specific issue.

**Comment by Soudini** [comment]
I am currently on a OnePlus 8T. I tried stable, beta and nightly. Out of the three, both stable and beta were running at 120fps but only nightly is running at 60fps (and only when a page is displayed, for example when typing a url, the frame rate goes up to 120fps according to the fps counter from the développer options).

I use only nightly because it is the only one working correctly with KeePassDX password auto completion integration. Is there anything I can do to help out ?

**Comment by dead10ck** [comment]
@Soudini I think this is an issue with OxygenOS, and not Firefox.

**Comment by Soudini** [comment]
You are totally right ! After doing some more research and buying AutoHz, it works. It's a shame that buying a third-party app is needed, but it is not this project's fault.

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1795138

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
